### PR TITLE
improve clarity on data download requirements

### PR DESCRIPTION
**Pull request by robcaulk**
We have had confusion in the main discord regarding how much data should be pre-downloaded for backtesting. This PR aims to help the lost souls who plan to never read it ;)

### CODE REVIEW

File: docs/freqai.md [hunk 1]
```diff
@@ -515,10 +515,11 @@ and if a full `live_retrain_hours` has elapsed since the end of the loaded model
 The FreqAI backtesting module can be executed with the following command:
 
 ```bash
-freqtrade backtesting --strategy FreqaiExampleStrategy --config config_examples/config_freqai.example.json --freqaimodel LightGBMRegressor --timerange 20210501-20210701
+freqtrade download-data --config config_examples/config_freqai.example.json --strategy FreqaiExampleStrategy --strategy-path freqtrade/templates -t 3m 15m 1h --timerange 20210315-20210701
```
Comment (xmatthias) [inline docs/freqai.md]: wouldn't a link (and a comment saying we assume data is available) to the downloading docs suffice?

i'm worried about having this here as people will assume it's always required - which it clearly isn't (you only need this the first time).

apart from that - the command is wrong - download-data ignores the end-date in such a case.

Comment (robcaulk) [reply docs/freqai.md]: It is a fair point. Link + note will do

Comment (xmatthias) [reply docs/freqai.md]: they'll not read it anyway :laughing:

**Review by xmatthias — APPROVED** [review]
Great :+1:

**Pull request merged**